components/PostCardList/PostCardListItemLoader.tsx

```diff
@@ -0,0 +1,176 @@
+import * as React from "react";
+import ContentLoader from "react-content-loader";
+import styled from "styled-components";
+import Card from "../Card";
+
+interface Props extends React.Attributes {
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PostCardListItemLoader({ ...props }: Props) {
+  return (
+    <Card {...props}>
+      <Loader>
+        <HeroHand />
+
+        <TitleFirstLine />
+        <TitleSecondLine />
+        <TitleThirdLine />
+
+        <LikeIcon />
+        <LikeNumber />
+
+        <PlayAtTitle />
+        <EndedAtTitle />
+        <FinalPotTitle />
+        <PostedTitle />
+
+        <PlayAtValue />
+        <EndedAtValue />
+        <FinalPotValue />
+        <PostedValue />
+      </Loader>
+    </Card>
+  );
+}
+
+const Loader = styled(ContentLoader)`
+  --hero-hand-diameter: 96px;
+  --margin-right-of-hero-hand: 16px;
+  --title-height: 18px;
+  --title-line-height: calc(var(--title-height) * 1.6);
+  --like-icon-size: 24px;
+  --attribute-height: 14px;
+  --attribute-width: 15%;
```

Comment:
I couldn't find where this value `15%` comes from. Where does this value refer from?